server/src/main/java/org/apache/druid/server/coordinator/CoordinatorDynamicConfig.java

```diff
@@ -127,8 +123,6 @@ public CoordinatorDynamicConfig(
   {
     this.markSegmentAsUnusedDelayMillis =
         markSegmentAsUnusedDelayMillis;
-    this.mergeBytesLimit = mergeBytesLimit;
```

Comment:
Curious - were these configs never used, or was their actual usage removed at some point, leaving just the configs in the code?  

Also, I see these configs are referenced in a few places like the docs and web-console: https://github.com/apache/druid/blob/master/docs/configuration/index.md#dynamic-configuration, https://github.com/apache/druid/blob/master/docs/api-reference/dynamic-configuration-api.md, https://github.com/apache/druid/blob/master/web-console/src/druid-models/coordinator-dynamic-config/coordinator-dynamic-config.tsx#L33